apps/fl-exp-simple/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "simple",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "build": "npm run synth",
+    "deploy": "cdk deploy",
+    "local": "fl local",
+    "synth": "cdk synth",
+    "test": "jest",
+    "validate": "functionless"
+  },
+  "devDependencies": {
+    "@aws-cdk/aws-appsync-alpha": "^2.41.0-alpha.0",
+    "@functionless/ast-reflection": "^0.3.1",
+    "@functionless/language-service": "^0.0.4",
+    "@types/jest": "^29.0.2",
+    "@types/node": "^18.7.18",
+    "@types/uuid": "^8.3.4",
+    "aws-cdk": "*",
+    "aws-cdk-lib": "*",
+    "aws-sdk": "^2.1216.0",
+    "constructs": "^10.1.111",
+    "esbuild": "^0.15.7",
+    "fl-exp": "*",
+    "functionless": "^0.23.1",
```

Comment:
Should be `*` to pick up workspace.

Reply:
All of them or just `functionless`?

Reply:
Oh i see, these are devDependencies. Looks like we're missing some `dependencies` in fl-exp.

Reply:
Adding this peer to `fl-exp`

```
  "peerDependencies": {
    "functionless": "^0.24.0"
  },
```
  
Will rely on transitive peers from `functionless` for the rest.

Reply:
Actually, why isn't `functionless` a dependency and not a peer?

Reply:
Changing `constructs: "*"` caused a ton of type errors. Had to pin everything to `10.0.0` or else i would end up with `10.111.0` in the root `node_modules` and `10.0.0` in the child `node_modules` causing type errors.

Reply:
Based on my experience, `*`, causes problems with sharing dependencies. Pattern that I see working is to use the same fixed version in `devDependencies`, e.g. `esbuild: "0.15.9"` and then specify a range in the peer dependency, e.g. `^0.15.9`. Using `*` results in the latest version always being installed in the root.